### PR TITLE
Adding Images app

### LINKED ISSUE #38: Add Images app
Add an Images table to the professionals.

### PR DESCRIPTION
**Pull request by idoyek**
I created the Images model.

The model has a foreign key from the Professional app, which hasn't been created yet while creating this PR, so for now it has been added within a comment.

When we add the Professional app, I'll make the necessary changes in the models.py.

In addition, I added the option to add images to the app, save it in a folder and giving its url to the ImageField model.
I followed this tutorial: https://www.youtube.com/watch?v=O5YkEFLXcRg

Resolves #38

### CODE REVIEW

**Review by guybeckenstein — APPROVED** [review]
LGTM

**Review by OfirBachar — APPROVED** [review]
LGTM

File: proImages/tests.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# from django.test import TestCase
```
Comment (Yarboa) [inline proImages/tests.py:1]: Please remove file

File: proImages/views.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# from django.shortcuts import render
```
Comment (Yarboa) [inline proImages/views.py:1]: Please remove file

File: setup.sh [hunk 3]
```diff
@@ -9,6 +9,7 @@ sudo -n dnf install -y pipenv
 cd /vagrant
 # Install dependencies with Pipenv
 pipenv sync --dev
+pipenv install pillow
```
Comment (Yarboa) [inline setup.sh:12]: I think you should upstade the Pipefile and Pipefile.lock
And add it once you made pipenv lock, please review tool introduction challenge

**Review by Yarboa — CHANGES_REQUESTED** [review]

**Review by Yarboa — COMMENTED** [review]
You can avoid filling the ImageFile in dest_data migration script
Please rebase and push from your terminal

**Review by idosinger98 — COMMENTED** [review]
Please create a function that receives as input Professional_id and returns as output an array of the images he uploaded

File: proImages/views.py (hunk 2 above)
Comment (idoyek) [reply proImages/views.py]: Done

File: setup.sh (hunk 3 above)
Comment (idoyek) [reply setup.sh]: Line removed from setup.sh

Comment (Yarboa) [reply setup.sh]: Please apply pipenv lock
And add Pipfile.lock to PR

Comment (idoyek) [reply setup.sh]: I already updated the Pipfile and Pipfile.lock files (I think it did it automatically when I installed Pillow), are more changes needed to be applied?

File: proImages/models.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class Images(models.Model):
+    images_ID = models.BigAutoField(primary_key=True)
```
[the remark this answers is not in the document]
Comment (dkeler) [reply proImages/models.py:5]: Also the ImageField is called `image`, in the singular form

File: proImages/models.py [hunk 5]
```diff
@@ -4,7 +4,7 @@
 class Images(models.Model):
     images_ID = models.BigAutoField(primary_key=True)
     # professional_ID = models.ForeignKey(Professional, on_delete=models.CASCADE)
```
Comment (Yarboa) [inline proImages/models.py:6]: Either remove this, and open Issue once the pro model exist to add it 
You can replace the pro_id with String

**Review by Yarboa — COMMENTED** [review]
still not test_data
https://github.com/redhat-beyond/Pro4U/pull/34#issuecomment-1511295525

File: proImages/models.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (idoyek) [reply proImages/models.py]: Fixed

File: proImages/models.py (hunk 5 above)
Comment (idoyek) [reply proImages/models.py]: Replaced it with a dummy professional_id

**Comment by idoyek** [comment]
> still not test_data [#34 (comment)](https://github.com/redhat-beyond/Pro4U/pull/34#issuecomment-1511295525)

Added now @Yarboa

**Review by idosinger98 — APPROVED** [review]
LGTM
Please open new Pr that include tests.py and function that receives as input Professional_id and returns as output an array of the images he uploaded

**Review by Yarboa — COMMENTED** [review]
@idoyek 
what about pytest tests? for the model? 
one or two is enough

**Comment by idoyek** [comment]
> @idoyek what about pytest tests? for the model? one or two is enough

I already did some, I'll upload it soon in a new PR

**Comment by idoyek** [comment]
> @idoyek what about pytest tests? for the model? one or two is enough

Done @Yarboa

**Review by TalReinfeld — APPROVED** [review]
LGTM

**Review by idosinger98 — APPROVED** [review]
LGTM

**Review by patrisiakaplun — APPROVED** [review]
LGTM

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**